[BUG] Unit test fail for unstructured io

### Required prerequisites

- [X] I have searched the [Issue Tracker](https://github.com/camel-ai/camel/issues) and [Discussions](https://github.com/camel-ai/camel/discussions) that this hasn't already been reported. (+1 or comment there if it has.)
- [ ] Consider asking first in a [Discussion](https://github.com/camel-ai/camel/discussions/new).

### Motivation
```
=========================== short test summary info ============================
FAILED examples/test/test_unstructured_io_example.py::test_parse_file_example - RuntimeError: /onnxruntime_src/onnxruntime/python/onnxruntime_pybind_state.cc:1406 onnxruntime::python::addGlobalMethods(pybind11::module&)::<lambda(int)> severity >= 0 && severity <= 4 was false. Invalid logging severity. 0:Verbose, 1:Info, 2:Warning, 3:Error, 4:Fatal
FAILED examples/test/test_unstructured_io_example.py::test_parse_url_example - RuntimeError: /onnxruntime_src/onnxruntime/python/onnxruntime_pybind_state.cc:1406 onnxruntime::python::addGlobalMethods(pybind11::module&)::<lambda(int)> severity >= 0 && severity <= 4 was false. Invalid logging severity. 0:Verbose, 1:Info, 2:Warning, 3:Error, 4:Fatal
FAILED examples/test/test_unstructured_io_example.py::test_stage_data_example - RuntimeError: /onnxruntime_src/onnxruntime/python/onnxruntime_pybind_state.cc:1406 onnxruntime::python::addGlobalMethods(pybind11::module&)::<lambda(int)> severity >= 0 && severity <= 4 was false. Invalid logging severity. 0:Verbose, 1:Info, 2:Warning, 3:Error, 4:Fatal
FAILED examples/test/test_unstructured_io_example.py::test_chunk_url_content_example - RuntimeError: /onnxruntime_src/onnxruntime/python/onnxruntime_pybind_state.cc:1406 onnxruntime::python::addGlobalMethods(pybind11::module&)::<lambda(int)> severity >= 0 && severity <= 4 was false. Invalid logging severity. 0:Verbose, 1:Info, 2:Warning, 3:Error, 4:Fatal
============ 4 failed, 37 passed, 11 warnings in 136.46s (0:02:16) =============
```

### Solution

_No response_

### Alternatives

_No response_

### Additional context

_No response_

--- a/camel/toolkits/human_toolkit.py
+++ b/camel/toolkits/human_toolkit.py
@@ -18,18 +18,6 @@ from typing import List
 from camel.toolkits.base import BaseToolkit
 from camel.toolkits.function_tool import FunctionTool
 
-# Add a new logging level called NOTICE, which is the same as WARNING
-# Used for logging questions, and it will be shown in the console
-NOTICE_LEVEL = 30  # same as WARNING
-logging.addLevelName(NOTICE_LEVEL, "NOTICE")
-
-
-def notice(self, message, *args, **kwargs):
-    if self.isEnabledFor(NOTICE_LEVEL):
-        self._log(NOTICE_LEVEL, message, args, **kwargs)
-
-
-logging.Logger.notice = notice  # type: ignore[attr-defined]
 logger = logging.getLogger(__name__)
 
 
@@ -48,7 +36,8 @@ class HumanToolkit(BaseToolkit):
         Returns:
             str: The answer from the human.
         """
-        logger.notice(f"Question: {question}")  # type: ignore[attr-defined]
+        print(f"Question: {question}")
+        logger.info(f"Question: {question}")
         reply = input("Your reply: ")
         logger.info(f"User reply: {reply}")
         return reply

